tests/trogdor/trogdor_runner.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+#
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import json
+import sys
+from produce_spec_runner import execute_produce_spec
+from trogdor_utils import update_trogdor_error
+from trogdor_utils import trogdor_log
+
+parser = argparse.ArgumentParser(description='Python Trogdor Producer.')
+parser.add_argument('--spec', dest='spec', required=False)
+args = parser.parse_args()
+
+
+def execute_task(workload):
+    try:
+        spec_class = workload["class"]
+        if spec_class == 'org.apache.kafka.trogdor.workload.ProduceBenchSpec':
+            execute_produce_spec(workload)
+        else:
+            update_trogdor_error(spec_class + " is not supported.")
+    except Exception as exp:
+        update_trogdor_error("Exception: " + exp)
+
+
+def get_task_from_specfile(specFile):
+    """
+    :param specFile: One Trogdor ExternalCommandSpec JSON file
+     One example of the ExternalCommandSpec spec:
+
+    :return specFile["spec"]["workload"] or None
+    """
+    specString = ""
+    workload = None
+    with open(specFile) as f:
+        for line in f:
+            if line.startswith('#'):
```

Comment:
suggest whitespace trimming the line

Reply:
I guess you had a specific reason to want to add support for comments, but i don't see this as critical, and since JSON doesn't support comments I think we should just leave this out completely.